src/main.ts

```diff
@@ -87,13 +100,19 @@ async function run(): Promise<void> {
       .map(file => file.filename)
       .filter(filename => filesToWatch.includes(filename))
     const prCompliant =
-      bodyCheck && titleCheck && branchCheck && filesFlagged.length === 0
+      bodyCheck &&
+      issueLabelErrors.length === 0 &&
+      titleCheck &&
+      branchCheck &&
+      filesFlagged.length === 0
     const shouldClosePr =
       (bodyCheck === false && bodyAutoClose === true) ||
-      (branchCheck === false && protectedBranchAutoClose === true)
+      (branchCheck === false && protectedBranchAutoClose === true) ||
+      (!!issueLabelErrors.length && issueLabelsAutoClose === true)
     // Set Output values
     core.setOutput('body-check', bodyCheck)
     core.setOutput('branch-check', branchCheck)
+    core.setOutput('issue-check', !!issueLabelErrors.length)
```

Comment:
would prefer naming to "issue-label-check". hard for me to think right now of what other issue checks might be appropriate, but i feel like it would be better to namespace it a bit more. Even bringing it into similar naming as what you have in README configuration would be good.
also, same question about length comparison as above.